annotation-file-utilities/src/annotator/find/ASTPathCriterion.java

```diff
@@ -96,9 +95,7 @@ public boolean isSatisfiedBy(TreePath path, Tree leaf) {
     /** {@inheritDoc} */
     @Override
     public boolean isSatisfiedBy(TreePath path) {
-        if (path == null) {
-            return false;
-        }
+        if (path == null || astPath.isEmpty()) { return false; }
```

Comment:
Please don't put actions on same line as conditions.  It's harder to read, it is not the standard Java style, and it is inconsistent with elsewhere in the file.  Furthermore, it makes the diffs larger.